src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -247,6 +239,7 @@ public void feedData(BibDatabaseContext bibDatabaseContext) {
         cleanUp();
 
         this.bibDatabaseContext = Objects.requireNonNull(bibDatabaseContext);
+        stateManager.setActiveDatabase(bibDatabaseContext);
```

Comment:
Context: The side pane is bound to the activeDatabaseProperty.

When an existing library is opened, a library tab will be added instantly with a loading animation. Then, the activeDatabaseProperty will be updated because a new tab was added and selected. The problem is at that time, the bibDatabaseContext has no entries because they are still loading. What I did here is set the REAL bibDatabaseContext as active after all entries are loaded.

Reply:
Please put it as Java Comment? 😇

Reply:
Thank you for pointing that out. I rephrased it a bit to fit better in a comment, but it should have the same meaning.